README.md

```diff
@@ -1,6 +1,5 @@
 # [Traileremma](https://willberner.github.io/Trailr/)
 
-A basic web application for searching, filtering, and sorting movies. Created with data pulled from the OMDb API and styled with Materialize.
```

Comment:
Forgot to delete this last time I pushed with a new description, but it's actually redundant now.